src/libraries/Microsoft.NETCore.Platforms/src/runtime.json

```diff
@@ -11,6 +11,12 @@
         "linux-musl-arm"
       ]
     },
+    "alpine-armv6": {
+      "#import": [
+        "alpine",
+        "linux-musl-armv6"
```

Comment:
In case it was not obvious, armv6 is only supported with mono runtime. From coreclr, we only build PAL, HP libunwind among few other pieces to support corehost (`dotnet(1)`) with `-p:RuntimeFlavor=mono`.

If this is working on Alpine Linux, I think would be nice to add a CI leg for `linux-musl-armv6` in extra-platforms pipeline (it may need some extra work in our cross infra setup). If this is untested, I think we should skip `armv6` for now.

cc @akoeplinger

Reply:
Mono is currently broken on musl due to using libc symbols, but I'm in the process of fixing it for `s390x` and `armv6`.